Add etherscan instance test for chain 42170

Etherscan instance tests are failing due to this error:

```
2) Test each Etherscan instance
       Double check that all supported chains are tested
         should have tested all supported chains:
     AssertionError: There are untested supported chains!: Arbitrum Nova (42170)
      at Context.<anonymous> (test/chains/etherscan-instances.spec.ts:59:12)
      at processImmediate (node:internal/timers:483:21)
      at process.topLevelDomainCallback (node:domain:160:15)
      at process.callbackTrampoline (node:internal/async_hooks:128:24)
```

--- a/services/server/test/helpers/etherscanInstanceContracts.json
+++ b/services/server/test/helpers/etherscanInstanceContracts.json
@@ -512,5 +512,22 @@
       "type": "standard-json",
       "expectedStatus": "partial"
     }
+  ],
+  "42170": [
+    {
+      "address": "0x0571d3aF023Ea8eD2323758B80Dd0C8DAe2C94f4",
+      "type": "single",
+      "expectedStatus": "partial"
+    },
+    {
+      "address": "0xa7016847E0285f15bf4449a2a260b379eB517100",
+      "type": "multiple",
+      "expectedStatus": "partial"
+    },
+    {
+      "address": "0xD779479d792Ab8E94f61d6b2682a877B2868A8AD",
+      "type": "standard-json",
+      "expectedStatus": "perfect"
+    }
   ]
 }

